UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/GetBatchDetails.cs

```diff
@@ -73,12 +72,31 @@ public static void GetBatchDetailsResponseValidationForFullAVCSExchangeSet(dynam
                 for (int responseFileNameLocation = 0; responseFileNameLocation < expectedFileName.Count; responseFileNameLocation++)
                 {
                     string responseFileName = batchDetailsResponse.files[responseFileNameLocation].filename;
-                    Assert.That(responseFileName, Is.EqualTo(expectedFileName[responseFileNameLocation]), $"Expected Response File Name of {expectedFileName[responseFileNameLocation]}, but actual value is {responseFileName}");
+                    responseFileName.Should().Be(expectedFileName[responseFileNameLocation]);
                 }
             }
             else
             {
-                Assert.Fail($"{mediaType} is different then Zip & DVD");
+                mediaType.Should().ContainAny("Zip","DVD");
+            }
+        }
+
+        public static void GetBatchDetailsResponseValidationForCatalogueXmlOrEncUpdateListCsv(dynamic batchDetailsResponse)
```

Comment:
Duplication of code, this can be simplified as follows:

            string responseContent = batchDetailsResponse.attributes[5].value;
            string responseFileName = batchDetailsResponse.files[0].filename;

            switch (responseContent)
            {
                case "Catalogue":
                    responseFileName.Should().Be(posDetails.AVCSCatalogueFileName);
                    break;
                case "ENC Updates":
                    responseFileName.Should().Be(posDetails.EncUpdateListFileName);
                    break;
                default:
                    responseContent.Should().ContainAny("Catalogue.xml", "Enc Update list.csv");
                    break;
            }